docs/source/generics.rst

```diff
@@ -370,6 +370,36 @@ type variables defined with special keyword arguments ``covariant`` or
    my_box = Box(Cat())
    look_into(my_box)  # OK, but mypy would complain here for an invariant type
 
+Note that the only change permitted in the variance of the generic class type
+variable as you move down the inheritance hierarchy is from covariant or
+contravariant to invariant. For example, this is valid:
+
+.. code-block:: python
+
+    from typing import Generic, TypeVar, Tuple, Iterable
+
+    K = TypeVar('K')
```

Comment:
It looks like this definition of type variable and the one below should be exchanged.